core/lib/zksync_core/src/lib.rs

```diff
@@ -5,6 +5,7 @@ use std::{net::Ipv4Addr, str::FromStr, sync::Arc, time::Instant};
 use anyhow::Context as _;
 use futures::channel::oneshot;
 use prometheus_exporter::PrometheusExporterConfig;
+use state_keeper::SequencerSealer;
```

Comment:
Nit: This is not an external import; it should be prefixed with `crate::` to make this clear.

Reply:
Fixed. Kinda sad that `rustfmt` can't do that automatically, since looks like `rust-analyzer` autoimport functionality is somewhat random.